optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/ScoreDirectorFactoryFactory.java

```diff
@@ -139,6 +144,29 @@ public ScoreDirectorFactoryFactory(ScoreDirectorFactoryConfig config) {
         return scoreDirectorFactory;
     }
 
+    protected String getDrlResourcesAndFilesOverview() {
```

Comment:
It's not a simple getter and it's for a toString, so how about we call it `composeScoreDrlAndScoreDrlFileToString`?

It might not be clear for everyone that DrlResourcesAndFiles applies to scoreDrl and scoreDrlFile.

Reply:
Update: I think we can remove this entire method if we add `ConfigUtils.abbreviate(List<String> list)` which calls `ConfigUtils.abbreviate(list, 3)` for which 3 is the limit size.

Reply:
Sure thing; I like the idea of reposrting the `DrlList` and `DrlFileList` separately, which makes this method obsolete.